nms/app/components/lte/LteContext.tsx

```diff
@@ -428,8 +428,25 @@ export function SubscriberContextProvider(props: Props) {
             key,
             value,
             newState,
-            newSessionState,
           }),
+        refetchSessionState: (id?: SubscriberId) => {
```

Comment:
A bit shorter would be:
```
refetchSessionState: (id?: SubscriberId) => {
     void FetchSubscriberState({networkId, id}).then(subscriberState => {
        if(subscriberState) {   
            setSessionState(sessionState => id ? {
              ...sessionState,
              [id]: subscriberState,
            } : subscriberState);
        }
     });
}
```

We could also cleanup `FetchSubscriberState` a bit so that it returns `{ [id]: subscriberState }` then it would have a consistent return type. Although be aware that we would still need to do   
`setSessionState(currentState => id ? {...currentState, ...subscriberState} : subscriberState)` Because we never want to merge the full new state with current state otherwise it would not be possible to delete entries.